docs/API.md

```diff
@@ -3,143 +3,134 @@ title: Getting Started
 permalink: /api/
 ---
 
-## Starting off
-First of all, you need to initialize the player.
+## Getting Started
+First of all, you need to set up the player. This usually takes less than a second:
```

Comment:
I think originally it had meaning that the call itself is taking less than a second (and resolves instantly if it was already initialized), and after this change it looks more like 'it's easy to setup'

Reply:
Yes, the issue here for me was the difference between getting started and getting into implementation details.. would it make sense to move the instant resolving part to the setupPlayer function documentation?